### ISSUE TITLE
How can I create with many Fields?

### ISSUE THREAD

**Issue by gelodefaultbrain** [kerrickstaley/genanki#50]
Hello first of all thank you with this python library. It is so cool! really helps me a lot in creating anki decks!

I have a question thoooo can you please help me and I think my question is just very very simple so here it is

```
my_note = genanki.Note(
  model=my_model,
  fields=['Capital of Argentina', 'Buenos Aires'])
```

On this code is you create a card right with the question "Capital of Argentina" and as the answer "Buenos Aires"... and it looks like this
![image](https://user-images.githubusercontent.com/55988471/94001423-14f4d700-fdcb-11ea-98f5-4f9da6eeb193.png)
![image](https://user-images.githubusercontent.com/55988471/94001460-1de5a880-fdcb-11ea-999c-07f7fae3150b.png)

I was thinking of can you provide an example on how can I make a field like this?
![image](https://user-images.githubusercontent.com/55988471/94001536-3655c300-fdcb-11ea-83c3-15ae703d93c9.png)

Thank you so much!

**Comment by remiberthoz** [comment]
Hey! I'm not the author or genanki but I can help on this one. When creating a note, you can pass as many fields as you wish:

```python
my_note = genanki.Note(
  model=my_model,
  fields=['Capital of Argentina', 'Buenos Aires', 'Casa Rosada'])
```

but this only makes sense if the model (`my_model`) you create defines the correct amount of fields:

```python
my_model = genanki.Model(
  1607392319,
  'Capital Model',
  fields=[
    {'name': 'Question'},
    {'name': 'City'},
    {'name': 'IconicMonument'},
  ],
  templates=[
    {
      'name': 'Card 1',
      'qfmt': '{{Question}}',
      'afmt': '{{FrontSide}}<hr id="answer">{{City}}<br>{{IconicMonument}}',
    },
  ])
```

So in you case you would have to define quite a lot of such fields when creating the model, and pass the list of values when creating notes.

**Comment by gelodefaultbrain** [comment]
Woooow Thank you so much! :D Got it! Also I'll be opening up a new issue... I think I got another one I don't know if this is intentional but I'll open it up in a bit... Thanks!

**Comment by kerrickstaley** [comment]
Thanks for the help on this, @remiberthoz !

I think one thing to help make this issue clearer is we could throw an exception when the number of `fields` in a `Note` doesn't match the number of `fields` in the `Model`. @gelodefaultbrain would that have helped you understand this problem?

**Comment by gelodefaultbrain** [comment]
Yeeep! Thank you so much!

**Comment by kerrickstaley** [comment]
I pushed https://github.com/kerrickstaley/genanki/commit/d3a5ac5d5a06ee29565a1173a1f63bbba252d265 which should help clarify this issue in the future.

**Comment by kerrickstaley** [comment]
Released in 0.8.1.
